src/components/ToolMenu/FeatureInfo/FeatureInfoForm/index.less

```diff
@@ -0,0 +1,18 @@
+.feature-info-form {
+  .ant-input-number,
+  .ant-picker {
+    width: 100%;
+  }
+
+  .ant-form-item {
+    .ant-row {
+      .ant-col.ant-form-item-label {
+        margin-right: 5px;
+        label {
+          word-break: break-word;
+          hyphens: auto;
+        }
+      }
+    }
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
just inserts a final newline. I'd recommend a plugin like [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) for this